src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -317,23 +317,23 @@
             *@
             <div class="form-group col-md-6 mb-0">
                 <label class="mb-0 ml-1"><small>Revision:</small></label>
-                <select class="selectpicker show-tick show-menu-arrow shadow-sm" data-style="btn-light btn-sm border" data-selected-text-format="value" data-live-search="true" data-size="10" data-width="100%" data-container="body" id="revisions-bootstraps-select" data-tick-icon="fa-solid fa-check">
+                <select class="custom-select shadow-sm custom-select-sm" id="revisions-bootstraps-select">
                     @foreach (var revision in Model.Review.Revisions.Reverse())
                     {
-                        var approvedBadge = "<i class='fas fa-check-circle text-success ml-2'></i>";
-                        var optionName = revision.IsApproved ? $"{@revision.DisplayName} {@approvedBadge}" : @revision.DisplayName;
-                        var urlValue = @Url.ActionLink("Review", "Assemblies", new {
+                        var optionName = revision.IsApproved ? $"{@revision.DisplayName} (APPROVED)" : @revision.DisplayName;

```

Comment:
So it's going to show APPROVED label instead of green tick?

Reply:
Yes for now.